tests/storage/test_throughput.py

```diff
@@ -74,16 +79,19 @@ def write_test(self, number_of_files, input_size):
 
             except Exception as err:
                 self.log.write("error", "Error while writing " + file_name)
+                self.stats[self.fname].set_error(err)
                 self.log.write("error", file_name + ": " + str(err))
                 return 1
-
             else:
+
                 size, fsize = self.ops.convert_size(input_size)
+                self.stats[self.fname].set_success()
```

Comment:
If this structure is fine, I'll add these lines in other tests too.

Reply:
Uhm... It would work but I am not particularly happy with it.
I suggest we discuss this during the weekly call